stdlib/2/__builtin__.pyi

```diff
@@ -1539,8 +1546,13 @@ else:
     def round(number: SupportsFloat, ndigits: int) -> float: ...
 def setattr(__obj: Any, __name: Text, __value: Any) -> None: ...
 if sys.version_info >= (3,):
+    @overload
+    def sorted(__iterable: Iterable[_LT], *,
+               key: None = ...,
+               reverse: bool = ...) -> List[_LT]: ...
+    @overload
     def sorted(__iterable: Iterable[_T], *,
-               key: Optional[Callable[[_T], Any]] = ...,
+               key: Callable[[_T], _LT],
```

Comment:
Same here.

Reply:
I understand your point here too, but I think in this case using the type var helps with readability. The `_LT` is needed in the overload right above this one, therefore using it here may help people understand more easily what are the relevant differences between the two overloads. Reverting back to `_SupportsLessThan` in this second overload introduces a bit of noise, in my opinion.

I'd like to know what people think about this.

Reply:
I agree with @srittau here. A typevar used in a single position doesn't make semantic sense and type checkers could reasonably throw an error for code like this.

Reply:
But here there is no invariance problem, so the `_LT` can just be `_SupportsLessThan`.